airbyte-integrations/template/python-source/setup.py

```diff
@@ -0,0 +1,15 @@
+from setuptools import setup, find_packages
+
+setup(
+    name='template-python-source',
+    description='Source implementation template',
+    author='Airbyte',
+    author_email='contact@airbyte.io',
+
+    packages=find_packages(),
+    package_data={
+        '': ['catalog.json']
```

Comment:
I'd slightly prefer *.json so we don't imply the need of a hard coded catalog at the package level when someone looks here.